packages/edge-gateway/wrangler.toml

```
@@ -148,4 +142,8 @@ deleted_classes = ["GatewayRateLimits2"]
 [[migrations]]
 tag = "v5" # Should be unique for each entry
 new_classes = ["GatewayRateLimits4"]
-deleted_classes = ["GatewayRateLimits3"]
\ No newline at end of file
+deleted_classes = ["GatewayRateLimits3"]
+
+[[migrations]]
+tag = "v5" # Should be unique for each entry
```

Should be unique for each entry!
```suggestion
tag = "v6" # Should be unique for each entry
```